code/core/src/core-server/presets/common-manager.ts

```diff
@@ -40,6 +41,7 @@ const tagFiltersManager = addons.register(TAG_FILTERS, (api) => {
 });
 
 export default [
+  onboardingManager,
```

Comment:
Maybe we need to make this optional?

If that's indeed needed, we can go about it in a few ways.
Considering the UI code is already all lazy-loaded, maybe we can do it at runtime, or maybe we don't inject here, based on a global we inject via the `common-preset`?